package.json

```diff
@@ -3,14 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@reduxjs/toolkit": "^1.8.5",
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
     "prop-types": "^15.8.1",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-redux": "^8.0.2",
     "react-router-dom": "^6.3.0",
     "react-scripts": "5.0.1",
+    "redux": "^4.2.0",
     "web-vitals": "^2.1.4"
   },
```

Comment:
### Your Readme
![Screenshot from 2022-08-23 19-28-02](https://user-images.githubusercontent.com/93039359/186184588-9c4d3136-091f-494c-b4aa-932e5b73bd3d.png)

- [ ] Please add missing things in your readme file to make it more descriptive. Your readme file is the documentation of your project so please take it seriously :+1: You can use [microverse template](https://github.com/microverseinc/readme-template/blob/master/README.md) to make your readme descriptive :+1:

Reply:
Done

Reply:
Cool :blush: